contracts/commands/CmBasicBuyCommand.sol

```diff
@@ -28,6 +28,11 @@ import { McdView } from "../McdView.sol";
 import { CmBaseMPACommand } from "./CmBaseMPACommand.sol";
 import { AutomationBotAggregator } from "../AutomationBotAggregator.sol";
 
+/*
+Changes compared to BasicSellCommand
+1) Inherites from CmBaseMPACommand not BaseMPACommand and therefore uses different recreateTrigger function
+2) Has triggerType 5 instead of 3
```

Comment:
```suggestion
1) Inherits from CmBaseMPACommand, not BaseMPACommand and therefore uses different recreateTrigger function
2) Has triggerType 5, instead of 3
```